tests/x11/seahorse.pm

```diff
@@ -24,9 +24,18 @@ sub run {
     send_key "ctrl-n";                                # New keyring
     assert_screen "seahorse-keyring-selector";        # Dialog "Select type to create"
     assert_and_dclick "seahorse-password-keyring";    # Selection: Password keyring
-    assert_screen "seahorse-name-new-keyring";        # Dialog  "Add a password keyring; name it"
-    type_string "Default Keyring";                    # Name of the keyring
-    send_key "alt-o";                                 # &Ok
+    my @tags = qw(seahorse-name-new-keyring ok_on_top);
+    assert_screen \@tags;                             # "Add a password keyring; name it"
+                                                      # may be with ok buttom on top or bottom of popup
+    if (check_screen "ok_on_top") {
```

Comment:
Better use `match_has_tag` here, as we already did the assertion above, no need to do that second time.